### PR TITLE
s/fireeye/mandiant

### PR DESCRIPTION
**Pull request by mr-tz**
Rename all FireEye refs to Mandiant

WIP

### Checklist

- [x] No CHANGELOG update needed

- [x] No new tests needed

- [ ] No documentation update needed

### CODE REVIEW

**Review by williballenthin — APPROVED** [review]
thank you!

**Comment by mr-tz** [comment]
Would like another quick review on this PR, https://github.com/mandiant/capa-rules/pull/466, and https://github.com/mandiant/capa-testfiles/pull/108.
I hope the GitHub syncs etc. continue working after the domain / email updates.

ping @Ana06 @mike-hunhoff

**Comment by mr-tz** [comment]
https://github.com/mandiant/capa-testfiles/pull/108 worked, so we should be good 🤞

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,8 +18,8 @@
 ### Development
 
 ### Raw diffs
-- [capa <release>...master](https://github.com/fireeye/capa/compare/v3.0.2...master)
-- [capa-rules <release>...master](https://github.com/fireeye/capa-rules/compare/v3.0.2...master)
+- [capa v3.0.2...master](https://github.com/fireeye/capa/compare/v3.0.2...master)
+- [capa-rules v3.0.2...master](https://github.com/fireeye/capa-rules/compare/v3.0.2...master)
```
Comment (williballenthin) [inline CHANGELOG.md:22]: good catch, thank you

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ This release fixes an issue with the standalone executables built with PyInstall
 - fix bug in PyInstaller config preventing ELF analysis #795 @mr-tz
 
 ### Raw diffs
-- [capa v3.0.1...master](https://github.com/fireeye/capa/compare/v3.0.1...v3.0.2)
-- [capa-rules v3.0.1...master](https://github.com/fireeye/capa-rules/compare/v3.0.1...v3.0.2)
+- [capa v3.0.1...v3.0.2](https://github.com/fireeye/capa/compare/v3.0.1...v3.0.2)
+- [capa-rules v3.0.1...v3.0.2](https://github.com/fireeye/capa-rules/compare/v3.0.1...v3.0.2)
```
Comment (williballenthin) [inline CHANGELOG.md:34]: ugh, and again, thank you

**Review by williballenthin — APPROVED** [review]
looks great, thank you!

**Pull request merged**